testing/helpers/index.js

```diff
@@ -62,3 +62,54 @@ export async function awaitFrameAfter(ms) {
   await sleep(ms);
   await afterRenderPromise();
 }
+
+const VOID_ELEMENTS = new Set([
+  'AREA',
+  'BASE',
+  'BR',
+  'COL',
+  'EMBED',
+  'HR',
+  'IMG',
+  'INPUT',
+  'LINK',
+  'META',
+  'PARAM',
+  'SOURCE',
+  'TRACK',
+  'WBR',
+]);
+
+/**
+ * Returns true of node is a void element.
+ * @param {Node} node
+ * @return {boolean}
+ */
+function isVoidElement(node) {
+  return node.nodeType === Node.ELEMENT_NODE && VOID_ELEMENTS.has(node);
```

Comment:
`VOID_ELEMENTS.has(node)` will never be true iiuc. `node` is a `Node`, `VOID_ELEMENTS` holds strings